File: Exemplos/App_IntentDigitalHUB_Flutter/Elgin_IntentDigitalHUB_Flutter-font/lib/IntentDigitalHubService/TERMICA/imprime_imagem.dart
import 'package:intent_digital_hub/IntentDigitalHubService/TERMICA/termica_commad.dart';

class ImprimeImagem extends TermicaCommand {
  final String _path;

  ImprimeImagem(this._path) : super('ImprimeImagem');

  @override
  functionParameters() => '"path"' ':' '"$_path"';
}

File: Exemplos M10/App_IntentDigitalHUB_Kotlin/Elgin_IntentDigitalHUB_Kotlin-font/app/src/main/java/com/elgin/intent_digital_hub/IntentDigitalHubService/TERMICA/Commands/ImprimeImagem.kt
package com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.Commands

import com.elgin.intent_digital_hub.IntentDigitalHubService.TERMICA.TermicaCommand

class ImprimeImagem(private val path: String) : TermicaCommand("ImprimeImagem") {
    override fun functionParameters(): String {
        return "\"path\"" + ":" + "\"" + path + "\""
    }
}